stdlib/public/core/CompilerProtocols.swift

```diff
@@ -21,7 +21,6 @@
 /// Only three types provided by Swift, `Bool`, `DarwinBoolean`, and `ObjCBool`,
 /// conform to `BooleanType`. Expanding this set to include types that
 /// represent more than simple boolean values is discouraged.
-@swift3_migration(renamed="Boolean")
```

Comment:
Right, and we already have an unavailable declaration for the `BoleanType` [here](https://github.com/apple/swift/blob/master/stdlib/public/core/CompilerProtocols.swift#L687).